src/Shared/Recruit.Vacancies.Client/Infrastructure/StorageQueue/StorageQueue.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Threading.Tasks;
+using Esfa.Recruit.Vacancies.Client.Application.Queues;
+using Esfa.Recruit.Vacancies.Client.Application.Queues.Messages;
+using Esfa.Recruit.Vacancies.Client.Infrastructure.EventStore;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Queue;
+using Newtonsoft.Json;
+
+namespace Esfa.Recruit.Vacancies.Client.Infrastructure.StorageQueue
+{
+    internal class StorageQueue : IQueue
+    {
+        private static readonly Dictionary<Type, string> MessageToStorageQueueMapper = new Dictionary<Type, string> 
```

Comment:
Does this need to be static

Reply:
Does it not need to be static?

Reply:
It's really a private instance field that is in a non-static class being used by non-static methods from what I can see.

Reply:
My thinking is this does not need to change between instances. But will change as I don't mind either way 👍